scripts/soroban-settings/write_upgrade_bytes/src/lib.rs

```diff
@@ -0,0 +1,31 @@
+#![no_std]
+use soroban_sdk::{contract, contractimpl, contractmeta, Env, Bytes, BytesN};
+
+pub(crate) const BUMP_AMOUNT: u32 = 518400; // 30 days
+
+contractmeta!(
+    key = "v",
+    val = "1"
+);
+
+#[contract]
+pub struct WriteBytesContract;
+
+#[contractimpl]
+impl WriteBytesContract {
+    pub fn write(env: Env, xdr_bytes: Bytes) -> BytesN<32> {
+        let hash = env.crypto().sha256(&xdr_bytes);
+        env.storage().persistent().set(&hash, &xdr_bytes);
+        env.storage().persistent().bump(&hash, BUMP_AMOUNT);
+
+        env.storage().instance().bump(BUMP_AMOUNT);
```

Comment:
Should we maybe bump by max possible amount? I wonder if we want this to ever expire...

Reply:
So there's actually an issue here because the script uses a random salt, meaning every time we run the script a new contract is created. I'll change this to use a fixed salt and  bump to the max.

Reply:
I actually can see this go either way: we can have a short-lived contract  and never bump it, or we can have a long-lived contract and maintain it, which will save some latency on upgrade but OTOH is a maintenance burden... I think at least for the initial period it would be better to keep the instance around as we are going to do a sequence of upgrades initially.

Reply:
After thinking about this (and also wasting time trying to figure out how to check that the contract exists before creating it) I think it might make sense to use a new contract per upgrade for two reasons - 1. We won't have to care about the contract expiring, and 2. This might be less error prone since the contractID in the configupgradesetkey will change everytime. My only concern was that we'd like these instances expire, but that isn't really a big deal in this case since we won't have too many of these contracts.

Reply:
Yeah, that sounds good to me.